task 8.3/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace task_8._3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+                Console.WriteLine("Введите а");
+                var a = double.Parse(Console.ReadLine());
+                var n = 0;
+                var sum = 0.0;
+                while (sum < a)

```

Comment:
Ошибка в неравенстве
```suggestion
                while (sum <= a)

```